private/model/api/api.go

```diff
@@ -504,6 +526,7 @@ func newClient(cfg aws.Config, handlers request.Handlers, endpoint, signingRegio
     		cfg,
     		metadata.ClientInfo{
 			ServiceName: {{ ServiceNameValue . }},
+			ServiceID : {{ ServiceID . }},
```

Comment:
Would be helpful to render this as a const within the `server.go` file similar to `ServiceName`

Reply:
Are there any services without a service id?

Reply:
I can make it a const. Doesn't hurt at least.